src/categories/topics.js

```diff
@@ -175,4 +180,10 @@ module.exports = function (Categories) {
 		await Promise.all(promises);
 		await Categories.updateRecentTidForCid(cid);
 	};
+
+	async function filterScheduledTids(tids) {
+		const scores = await db.sortedSetScores('topics:scheduled', tids);
+		const now = Date.now();
+		return tids.filter((tid, index) => tid && scores[index] <= now);
```

Comment:
If topic isn't scheduled score will be null. This works but maybe a better check would be 
```
return tids.filter((tid, index) => tid && (!score[index] || scores[index] <= now));
```

Reply:
Done. :+1: